### ISSUE TITLE
Implementation and processing of V2G Message？

### ISSUE THREAD

**Issue by Baymax-Rob** [nvdungx/HLC_TestTool#2]
Hello nvdungx,
 I am very interested in this project. Thanks to the author for his contribution. I saw that HLC_Test_V2G_Message_Check.xlsx in your docs has not completed the CertificateUpdate/Installation Req/Res Message. Did you encounter any difficulties or problems?

**Comment by nvdungx** [comment]
Hi Rob,
I have not implemented the PnC related msg yet, but I think there should be no difficulties with those msgs since they are just a medium to send/receive certificate data between EVCC and SECC.
Assuming that we only validate the conformance of protocol operation, no interaction with a secondary actor is required. Test bench/environment can operate locally, provide OEM Cert(OEM provisioning certification installed on EVCC)  and act as root CA to create certification data directly for updating/installing to EVCC.

**Comment by Baymax-Rob** [comment]
Thank you very much for your reply, and thank you for the open source project. Because we are planning to design EVCC, we are learning through your project, although your project is related to this part of SECC. But since I have never touched this part before, it is a bit difficult at the beginning. We are currently familiar with ISO 15118-2/ISO 15118-3 and Ethernet, and C++ is still in the learning stage. If there is a chance, we would like to participate in your project.

**Comment by nvdungx** [comment]
Yes feel free to take reference from this project, but I'm currently in process of re-implementing 2nd version of this, and for ISO15118-3 as well, so this repos will not be updated. I will push the new one when it's somewhat available.
